Fruitties/shared/schemas/com.example.fruitties.database.AppDatabase/1.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
do we need a license on the top of this file?

Reply:
Usually they don't have license since they are generated and read back by Room.

Reply:
We cannot add a license to a JSON file, since it would break the file format. This is also a generated file whose purpose is to identify breaking schema changes, so we should not be editing it manually. In fact, this file should probably never change. Instead of changing this file in the future, best practice would involve updating the database version number, which would generate a new JSON file (like 2.json).

Reply:
ok sounds good, thanks!